components/badge/badge.css

```diff
@@ -0,0 +1,34 @@
+@import url(https://feather-com.netlify.app/styles.css);
+
+.badge-base{
+    font-size: large;
+    padding: 1rem;
+    width: fit-content;
+    position: relative;
+    color: var(--primary-color);
+    /* outline: red solid 1px; */
+}
+
+.badge{
+    position: absolute;
+    font-size: small;
+    padding: 0.3rem;
+    width: 1.2rem;
```

Comment:
You can remove the width attribute, padding takes care of the badge text being centered and and badge being circular